### DISCUSSION TITLE
Question about https://torstees.github.io/ncpi-fhir-ig-2/#high-level-picture

### DISCUSSION THREAD

**Discussion by mingward** [NIH-NCPI/ncpi-fhir-ig-2#33] [General]
Hi, Mike Feolo and I are looking at https://torstees.github.io/ncpi-fhir-ig-2/#high-level-picture.  what is the relationship(parity) between ResearchSubject and Patient? in the picture, it has an arrow.  In dbGaP, one Patient can be linked to multiple ResearchSubject. Thank you!

**Comment by torstees** [comment]
Hi there! Thanks for reaching out.

Robert created this diagram, but I believe the arrow is appropriate. Those ResearchSubject resources are just there to link the patient to the study. I imagine what you are describing with DbGAP would be a single study that has some number of substudies/consent groups. The complete set of patients may be linked to the top level study via one set of ResearchSubjects and each substudy would be linked to their respective members in a similar manner. This would allow patients to be linked to more than one substudy.

Also, we have updated the official IG V2 website with this and much more content. I highly recommend you replace the old bookmarks to use the one blow.

https://nih-ncpi.github.io/ncpi-fhir-ig-2/

Happy to chat more if this isn't clear.

Thanks,
eric
________________________________
From: mingward ***@***.***>
Sent: Thursday, August 15, 2024 1:38 PM
To: NIH-NCPI/ncpi-fhir-ig-2 ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [NIH-NCPI/ncpi-fhir-ig-2] Question about https://torstees.github.io/ncpi-fhir-ig-2/#high-level-picture (Discussion #33)


Hi, Mike Feolo and I are looking at https://torstees.github.io/ncpi-fhir-ig-2/#high-level-picture. what is the relationship(parity) between ResearchSubject and Patient? in the picture, it has an arrow. In dbGaP, one Patient can be linked to multiple ResearchSubject. Thank you!

—
Reply to this email directly, view it on GitHub<https://github.com/NIH-NCPI/ncpi-fhir-ig-2/discussions/33>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AAGQXQ7MKZ4A3G4MEY6QHG3ZRTYRNAVCNFSM6AAAAABMSWI3OKVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZXGA2TONBRGA>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

[ WARNING : This email came from an external source. Please treat this message with additional caution.]
